paywall/src/components/content/NewDemoContent.tsx

```diff
@@ -24,8 +24,10 @@ export default function NewDemoContent() {
   const locks: string[] = url.searchParams.getAll('lock')
   const names: string[] = url.searchParams.getAll('name')
   const type = url.searchParams.get('type')
+  const unlockUserAccounts = url.searchParams.get('unlockUserAccounts')
```

Comment:
Will this come back as `undefined` when this param isn't in the URL? Might be worth explicitly handling that so we always have a boolean here.

Reply:
It returns `null`!